exchanges/btse/btse_wrapper.go

```diff
@@ -248,7 +248,12 @@ func (b *BTSE) FetchTradablePairs(ctx context.Context, a asset.Item) (currency.P
 	}
 	pairs := make([]currency.Pair, 0, len(m))
 	for x := range m {
-		if !m[x].Active {
+		if !m[x].Active ||
+			// BTSE returns 0 for both highest bid and lowest ask if there is
```

Comment:
Just a note here: We'll need a TODO adding support for an OTC asset as this eliminates many valid tradable pairs which are active, OTC only and available on the front-end.